.github/workflows/workflow.yml

```diff
@@ -233,3 +233,17 @@ jobs:
         env:
           # We disable the Dune cache when running the tests
           DUNE_CACHE: disabled
+
+  monorepo_benchmark_test:
+    name: Build monorepo benchmark docker image
+    runs-on: ubuntu-latest
```

Comment:
Do we think that this is a good idea? I am worried that if `ubuntu-latest` switches to a new Ubuntu release, it'll potentially break down.

Reply:
Possibly, but note that the dockerfile itself specifies a very specific version (of debian) and its unlikely that when ubuntu gets updated it shouldn't affect how docker behaves.